easybuild/easyblocks/t/trilinos.py

```diff
@@ -124,6 +124,12 @@ def configure_step(self):
                 lib_names += ";libgfortran.a"
             self.cfg.update('configopts', '-D%s_LIBRARY_NAMES="%s"' % (dep, lib_names))
 
+        # MKL
+        if get_software_root('imkl'):
+            self.cfg.update('configopts', "-DTPL_ENABLE_MKL:BOOL=ON")
```

Comment:
@vanzod Maybe it's worth considering only doing this conditionally for sufficiently recent versions of Trilinos, e.g. `> 12.12` (which is already quite old, `12.12.1` dates from Sept'17)